Kickstarter-iOS/Views/Controllers/RewardsCollectionViewController.swift

```diff
@@ -190,13 +196,28 @@ final class RewardsCollectionViewController: UICollectionViewController {
     return CGSize(width: itemWidth, height: itemHeight)
   }
 
+  private func goToPledge(project: Project, reward: Reward, refTag: RefTag?) {
```

Comment:
Is there a reason why we are passing the `refTag` on this function? It's not being used here.

Reply:
Yeah we're definitely going to need to pass it to the `PledgeViewController`, so I added it in anticipation for that.